pkgs/applications/misc/tandoor-recipes/common.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub }:
+let
+  version = "1.3.3";
+  srcSha256 = "sha256-hnTDwnVTvVxSVzwUYBHBJpMqsvl/o011nkddYSJQQdQ=";
+  yarnSha256 = "sha256-HnFCz+okFqLddxyQ56NPVmW6Z/mFqVe/7Ihp+Mol12I=";
+in
+{
+  inherit version yarnSha256;
+
+  src = fetchFromGitHub {
+    owner = "TandoorRecipes";
+    repo = "recipes";
+    rev = version;
+    sha256 = srcSha256;
+  };
+
+
+  meta = with lib; {
+    homepage = "https://tandoor.dev/";
+    license = licenses.agpl3Only; # FIXME: sure about that?
```

Comment:
Would like a final word on this, see [this issue](https://github.com/TandoorRecipes/recipes/issues/238) for a discussion about licensing.

Reply:
@vabene1111 is this correct at the moment?

Reply:
the code above is fine with me, the license will stay as it is unless someone who really knows what he does helps me fix it. I have made clear what I want to achive in the issue. Thanks for making the nix package